CMakeLists.txt

```diff
@@ -264,6 +282,16 @@ ecbuild_add_option( FEATURE AIO
                     CONDITION ${AIO_FOUND}
                     DESCRIPTION "support for asynchronous IO")
 
+### PROJ support
+
+ecbuild_add_option( FEATURE PROJ
+                    DEFAULT OFF
+                    DESCRIPTION "support PROJ-based projections"
+                    REQUIRED_PACKAGES "PROJ VERSION 9.2" )
```

Comment:
Probably no need for "PROJ VERSION 9.2", just "PROJ 9.2", see CURL comment above.